rktl_game/index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
```

Comment:
Can we stick all the web stuff in a folder for better organization? Maybe just a folder called `web`?

Reply:
done